config/default/testRunner.conf.php

```diff
@@ -388,6 +388,10 @@
         ],
         'previous' => [
             'trigger' => 'K'
+        ],
+        'dialog' => [
+            'accept' => 'Enter',
+            'reject' => 'X'
```

Comment:
May be, in order to be consistent with other modals in tao, and especially since `<esc>` is only unavailable when the fullscreen is requested using the API (not when the user asks it) which is only for one customer.